runtime.go

```diff
@@ -394,9 +392,14 @@ func getSessionOrDefaultCreds() map[string]string {
 
 			result["key"] = creds.AccessKeyID
 			result["secret"] = creds.SecretAccessKey
-			result["sessiontoken"] = creds.SessionToken
+			if creds.SessionToken != "" {
+				result["sessiontoken"] = creds.SessionToken
+			} else {
+				delete(result, "sessiontoken")
```

Comment:
You have already removed `sessiontoken` from the initialization. Why is this needed?

Reply:
Good catch